### DISCUSSION TITLE
Issue vivadli browser volume 100

### DISCUSSION THREAD

**Discussion by crimson39** [File-New-Project/EarTrumpet#702] [Ideas]
Hi 
I have a problem with the volume setting in the vivaldi browser. After starting the browser, the volume is always set to 100. Only this application behaves like this.
BS

**Comment by riverar** [comment]
Hi @crimson39, EarTrumpet doesn't persist app volumes at this time; Windows may handle this for you but it's not reliable. Will mark this as a feature request.

**Comment by crimson39** [comment]
Thank you for your answer. BR

**Comment by LukasDoubleU** [comment]
I implemented this and other features over at my fork.
You can enable the saving of app volumes in the settings.

If you want you can test the pre-release:
https://github.com/LukasDoubleU/EarTrumpet/releases
